<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen_Camera">
     <property name="title">
      <string>Open Camera</string>
     </property>
     <addaction name="actionFakeVideo"/>
     <addaction name="actionPointGrey"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuOpen_Camera"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuVideo">
    <property name="title">
     <string>Video</string>
    </property>
    <addaction name="actionDisplay"/>
    <addaction name="actionOpenVideoFile"/>
    <addaction name="actionRecord"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionConfigure"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuVideo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionDisplay">
   <property name="text">
    <string>Display</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpenVideoFile">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionFakeVideo">
   <property name="text">
    <string>Fake Video</string>
   </property>
  </action>
  <action name="actionPointGrey">
   <property name="text">
    <string>Point Grey</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
